### ISSUE TITLE
On decryption (optionally) restore original file system structure

### ISSUE THREAD

**Issue by rwiesbach** [mvt-project/mvt#353]
When an iTunes Backup is decrypted, the hexadecimal names are used for the decrypted output files, even though the path (including filename) is shown in the log. Why not create the structure of decrypted files accordingly? If there is a good reason not to do so, it should at least optionally be possible to do so.

**Comment by Te-k** [comment]
Yeah that's fair, but I don't think we should support an analysis on unpacked backup, it would be quite challenging to do
